README.md

```diff
@@ -1345,6 +1345,7 @@ Table of Contents
   * [haikei.app](https://www.haikei.app/) - Haikei is a web app to generate unique SVG shapes, backgrounds, and patterns – ready to use with your design tools and workflow.
   * [Canva](https://canva.com) - Free online design tool to create visual content.
   * [Superdesigner](https://superdesigner.co) - A collection of free design tools to create unique backgrounds, patterns, shapes, images, and more with just a few clicks.
+  * [TeleportHQ](https://teleporthq.io/) - Low-code Front-end Design & Development Platform. TeleportHQ is the collaborative front-end platform with integrated UI development and content modelling tools. A powerful visual builder to create and publish your headless static websites instantly.
```

Comment:
Thanks, would you mind saying a bit about whats free?

Reply:
Sure thing.
The users can create up to 3 projects, which can be entire UI of websites/web applications.
Full editor capabilities.
Multiple pages and reusable components.
Real time collaboration with other contributors.
Templates that can be used for free.
Code export to multiple frameworks & CSS Styling libraries (React + CSSModules/StyleJSX/, Vue, Ancular, Preact, HTML + CSS)
Free hosting under teleporthq domain.

Regards,
Alex

Reply:
i mean in your description.

For example see this one:

```
* [Zeplin](https://zeplin.io/) — Designer and developer collaboration platform. Show designs, assets and styleguides. Free for 1 project.
```

Short and sweet just showing what the free feature is.

Reply:
Oh, got it. Will do it now.